app/experimenter/nimbus-ui/src/components/TableSummary/index.stories.tsx

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { mockExperimentQuery } from "../../lib/mocks";
+import AppLayout from "../AppLayout";
+import TableSummary from ".";
+import { RouterSlugProvider } from "../../lib/test-utils";
+
+storiesOf("components/TableSummary", module)
+  .add("filled out", () => {
+    const { data } = mockExperimentQuery("demo-slug");
+    return (
+      <Subject>
+        <TableSummary experiment={data!} />
+      </Subject>
+    );
+  })
+  .add("partially filled out", () => {
+    const { data } = mockExperimentQuery("demo-slug", {
+      probeSets: [],
+      channels: [],
+      proposedDuration: 0,
+    });
+    return (
+      <Subject>
+        <TableSummary experiment={data!} />
+      </Subject>
+    );
+  })
+  .add("missing all fields", () => {
```

Comment:
FWIW this should never happen... the "Results" link will be disabled later if any _required fields_ are missing, but checking for a null value for each piece and displaying that "not set" text seemed like the right move to guard against it as a final sanity check on the request page (or if a user chooses to hit it directly and we don't have a redirect set up for it).